web/sablony/blackarrow/prihlaska.xtpl

```diff
@@ -113,7 +113,7 @@
 
   <!----------------------------------------------------------------------------->
 
-  {covidSekce}
+  <!-- covidSekce (zakomentovaná) -->
```

Comment:
Nenecháme tu ten původní, jen zakomentovaný kód?
`<!-- {covidSekce} -->`

Reply:
Chtěl jsem, ale celé se to pak vygeneruje v html, jen zakomentované, proto jsem zvolil toto řešení. A nenašel jsem jak v tom xtpl komentovat jinak :/